src/pages/documentation/platform-assurance/tutorials/push-debug-view.md

```diff
@@ -52,68 +52,68 @@ Finally, at the bottom of the panel is a link that will open this specific app s
 
 The "Send Test Push" tab can be used to send a test message to your device.
 
-#### Message Composition
+#### Message composition
 
-There are several panes that can be configured to test different iOS and Android push features. Once configured, press the "Send Test Push Notification" button to send your message.
+There are several panes that can be configured to test different iOS and Android push features. Once configured, select **Send Test Push Notification** to send your message.
 
 ![Send Push](./assets/push-debug-view/send.png)
 
 ##### Message
 
-In the "Message" pane, you can provide a title and body for the message. The silent notification feature can be enabled here as well.
+In the `Message` pane, you can provide a title and body for the message. The silent notification feature can be enabled here as well.
```

Comment:
My understanding was that in general UI elements are usually not documented in bold, but can be highlighted for easier read. And that bold is used selectable/actionable UI elements, like buttons. @calebk1 can you confirm?

Reply:
Hmm. Generally, we bold to emphasize thing, but we don't necessarily have a set precedent to only bold for UI elements (since in the Platform docs, I bold for stuff like do **not** select this or On the **Add values** popover.)

So, I think bolding here would be fine for emphasis?

Reply:
Yeah, I think we (as the Platform team) kinda bypassed that suggestion on the Platform-side, since we bold for emphasis since it just visually looks better to read.

Reply:
Hmm. I'd say either just leave it as no code, bold, or quotes as my preferred order.

Reply:
Ok, I used bold.